service_plans/v1/service_plans_test.go

```diff
@@ -0,0 +1,132 @@
+package service_plans
+
+import (
+	"fmt"
+	"math/rand"
+	"strings"
+
+	"github.com/cloudfoundry-incubator/cf-performance-tests/helpers"
+	"github.com/cloudfoundry-incubator/cf-test-helpers/workflowhelpers"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("service plans", func() {
+	Describe("GET /v3/service_plans", func() {
+		Context("as admin", func() {
+			Measure("list all", func(b Benchmarker) {
+				workflowhelpers.AsUser(testSetup.AdminUserContext(), testConfig.BasicTimeout, func() {
+					helpers.TimeCFCurl(b, testConfig.BasicTimeout, "/v3/service_plans")
+				})
+			}, testConfig.Samples)
+			Measure(fmt.Sprintf("list with page size %d", testConfig.LargePageSize), func(b Benchmarker) {
+				workflowhelpers.AsUser(testSetup.AdminUserContext(), testConfig.LongTimeout, func() {
+					helpers.TimeCFCurl(b, testConfig.LongTimeout, fmt.Sprintf("/v3/service_plans?per_page=%d", testConfig.LargePageSize))
+				})
+			}, testConfig.Samples)
+		})
+		Context("as regular user", func() {
+			Measure("list all", func(b Benchmarker) {
+				workflowhelpers.AsUser(testSetup.RegularUserContext(), testConfig.LongTimeout, func() {
+					helpers.TimeCFCurl(b, testConfig.LongTimeout, "/v3/service_plans")
+				})
+			}, testConfig.Samples)
+		})
+	})
+
+
+	Describe("GET /v3/service_plans/:guid", func() {
+		Context("as admin", func() {
+			var servicePlanGUID string
+			BeforeEach(func() {
+				servicePlanGUIDs := helpers.GetGUIDs(testSetup.AdminUserContext(), testConfig, "/v3/service_plans")
+				Expect(servicePlanGUIDs).NotTo(BeNil())
+				servicePlanGUID = servicePlanGUIDs[rand.Intn(len(servicePlanGUIDs))]
+			})
+			Measure("list one", func(b Benchmarker) {
+				workflowhelpers.AsUser(testSetup.AdminUserContext(), testConfig.BasicTimeout, func() {
+					helpers.TimeCFCurl(b, testConfig.BasicTimeout, fmt.Sprintf("/v3/service_plans/%s", servicePlanGUID))
+				})
+			}, testConfig.Samples)
+
+		})
+		Context("as regular user", func() {
+			servicePlanGUID := ""
```

Comment:
Variable definition (e.g. `var servicePlanGUID string`) could be added to the parent `Describe`.